Voycar.Api.Web/Features/Members/Endpoints/Get/Personal/Response.cs

```diff
@@ -0,0 +1,22 @@
+namespace Voycar.Api.Web.Features.Members.Endpoints.Get.Personal;
+
+public class Response
+{
+    // Personal information
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public string Street { get; set; }
+    public string HouseNumber { get; set; }
+    public string PostalCode { get; set; }
+    public string City { get; set; }
+    public string Country { get; set; }
+    public DateOnly BirthDate { get; set; }
+    public string BirthPlace { get; set; }
+
+    // Contact information
+    public string Email { get; set; }
+    public string PhoneNumber { get; set; }
```

Comment:
```suggestion
    public string? PhoneNumber { get; set; }
```
Da `Member.PhoneNumber` auch nullable ist. Vermeidet Warning von Rider im Mapper (Zeile 32)

Reply:
Dann sollten wir das ? aus dem Entity Member am besten auch rausnehmen. Mache ich gleich mit

Reply:
das geht natürlich auch